plugins/modem-manager/fu-mm-device.c

```diff
@@ -59,6 +59,7 @@ struct _FuMmDevice {
 	 */
 	MMModemFirmwareUpdateMethod update_methods;
 	gchar *detach_fastboot_at;
+	gchar *branch_at;
```

Comment:
unused?

Reply:
Used to store the quirk value.
This is then used as AT command to get the branch.

Reply:
I misread the function, sorry for the noise!

Reply:
Ohh, then don't you need to g_free it in finalize?

Reply:
Oh yes, good catch!